controllers/ingressController.go

```diff
@@ -22,33 +23,29 @@ type info struct {
 	Leaseinfo models.Leases
 }
 
-type ingressess []string
-
 //ShowAllowedIngress ...
 func ShowAllowedIngress(c *gin.Context) {
 	User, _ := c.Get("User")
-	ns, count := "", 0
-	ns = c.Query("ns")
-	var myIngress []pkg.IngressList
-	namespaces := make(map[string]int)
-
-	log.Infof("Listing ingress in namespace %s for user %s\n", ns, User.(*models.Users).Email)
-	for kubeContext, kubeClient := range config.KubeClients {
-		clientset := kubeClient.ClientSet
-		myclientset := pkg.MyClientSet{Clientset: clientset}
-		data, _ := myclientset.GetIngresses(kubeContext, ns)
-		for _, ingress := range data {
-			if val, ok := namespaces[ingress.Namespace+":"+ingress.Name]; ok {
-				myIngress[val].Context = myIngress[val].Context + "," + ingress.Context
-				continue
-			}
-			namespaces[ingress.Namespace+":"+ingress.Name] = count
-			myIngress = append(myIngress, ingress)
-			count = count + 1
+	ns := c.Query("ns")
+	var data []pkg.IngressList
+
+	req := ingress_driver.ShowAllowedIngressRequest{
```

Comment:
shouldn't it be `ShowAllowedIngressOptions`?

Reply:
removed. wasnt getting used